spec/features/queue_items_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'spec_helper'
+
+feature "User interacts with queue" do
+  scenario "user adds and reorders videos in the queue" do
+    riddle = Fabricate(:user, full_name: "Riddle Bear")
+    comedies = Fabricate(:category, name: "Comedies")
+    monk = Fabricate(:video, category: comedies, title: 'Monk')
+    south_park = Fabricate(:video, category: comedies, title: 'South Park')
+    futurama = Fabricate(:video, category: comedies, title: 'Futurama')
+
+    sign_in(riddle)
+
+    click_video(monk)
+    expect(page).to have_content monk.title
+
+    click_link "+ My Queue"
+    expect(page).to have_content monk.title
```

Comment:
How is expecting that page will have `monk.title` content different than the one on line 14? By reading the feature spec, I would expect that clicking "+ My Queue" link would at least deliver a message like "your item was queued up".